cmd/dtab.go

```diff
@@ -124,7 +138,7 @@ var (
 				if err != nil {
 					return err
 				}
-				_, err = ctl.Update(name, dtabstr, namer.Version(dtabUpdateVersion))
+				_, err = ctl.Update(name, dtabstr, dtabJson, namer.Version(dtabUpdateVersion))
```

Comment:
It looks like I'm use namerd v7, so you're right there could be an incompatibility here https://github.com/BuoyantIO/linkerd-examples/blob/master/gob/k8s/namerd/rc.yml#L24